platform/spire/src/__main__.py

```diff
@@ -15,7 +15,7 @@
 import version
 import virt
 
-main_command = command.mux_map("invoke a top-level command", {
+main_command = command.Mux("invoke a top-level command", {
```

Comment:
There are a bunch of the commits between the method signature changes and this commit that aren't marked as being expected not to function, even though it seems like they wouldn't function due to the signature mismatches not yet being resolved.

I'm worried about that, because I could easily imagine landing on one of those during bisecting, and assuming that the commit works correctly. Could we mark those intermediate commits as not being expected to function?

Reply:
Oops, I think the commits are in the wrong order somehow...

Reply:
Actually - what is an example of such a commit?  I don't see any

Reply:
There was 7e8719ce4244ae8b4b46f300f3f3dc84109360a1, but that was covered by another comment I made. I certainly can't see the same thing I remember seeing anymore, which means either some sort of weird Git thing happened or my eyes are fake. In any case, this looks good in the latest rebase you've done.

Reply:
I think at least one of GitHub's views does not display the commits in order

Reply:
I went through via the Files Changed commit iteration thing, which I think is in topological order?

Reply:
:woman_shrugging: :crab: